suites/reef/rgw/tier-4-rgw.yaml

```diff
@@ -224,7 +224,6 @@ tests:
       desc: increase ulimit number of open files to 16384
       module: exec.py
       name: increase ulimit number of open files to 16384
-      polarion-id: CEPH-10362
```

Comment:
@hmaheswa , Polarian Id is removed here,
please have a look if it is removed by mistake.

Reply:
@ckulal 
it is not the right polarion actually
may be its added by mistake
we cannot have any polarion actually to that TFA fix test